projects/core/src/i18n/translation-chunk.service.ts

```diff
@@ -3,25 +3,31 @@ import { I18nConfig } from './config/i18n-config';
 
 @Injectable()
 export class TranslationChunkService {
-  constructor(protected config: I18nConfig) {}
+  private chunks: object = {};
+  constructor(protected config: I18nConfig) {
+    Object.keys(config.i18n.chunks).forEach(chunk => {
+      config.i18n.chunks[chunk].map(key => {
+        if (this.chunks.hasOwnProperty(key)) {
+          this.warnDuplicates(key);
+        }
+        this.chunks[key] = chunk;
+      });
+    });
+  }
 
   protected readonly KEY_SEPARATOR = '.';
 
   getChunkNameForKey(key: string): string {
     const mainKey = (key || '').split(this.KEY_SEPARATOR)[0];
-    const chunk = this.getChunkFromConfig(mainKey);
+    const chunk = this.chunks && this.chunks[mainKey];
 
     if (!chunk) {
       return mainKey; // fallback to main key as a chunk
     }
     return chunk;
   }
 
-  private getChunkFromConfig(mainKey: string): string {
-    return (
-      this.config.i18n &&
-      this.config.i18n.chunks &&
-      this.config.i18n.chunks[mainKey]
-    );
+  warnDuplicates(key: string): void {
+    console.warn(`Duplicated key (${key}) in the i18n config file.`);
```

Comment:
It would be good help the Customer understand that the problem is with the **chunks config** in particular (not the whole i18n config).

We should also warn only in non-producton env.

What do you think about informing which value is taken and which ignored (from two duplicated)?